README.md

```diff
@@ -1,22 +1,42 @@
+# Env variables
+```
+API_PORT={number, ex.3040}
+
+LANG={RUST|GO|NODE|JAVA}
+ETH_NODE_URL={infura or alchemy endpoint}
+DB_CONNECTION_URL={postgres connection string}
+INTEGRATED=
+```
+
 # Using the test tool
 
+## Set env variables
+```
+export API_PORT="8080"
+export DB_CONNECTION_URL="postgresql://localhost/postgres?user=postgres&password=password1234"
+export ETH_NODE_URL={infura or alchemy endpoint}
```

Comment:
You have already stated that it has to be an infura or alchemy endpoint above in the _#Env variables_  section and judging by the 2 rows above, this seems like an example setup. I would change the right side of this line to `https://goerli.infura.io/v3/b6e734b85b1f42fd94f9fddcbdf556ba`

Reply:
Yeah, it should be used as an example, I left it like that so that it's not directly "copy-paste"-able, because then you will spend some time wondering why it's not working with the given values.
I don't mind setting it to `https://goerli.infura.io/v3/b6e734b85b1f42fd94f9fddcbdf556ba`, users of the tool just need to be mindful about that.

Reply:
Updated.